types/src/language_storage.rs

```diff
@@ -1,144 +1 @@
-// Copyright (c) The Libra Core Contributors
-// SPDX-License-Identifier: Apache-2.0
 
```

Comment:
is this file gone? I think it's missing, given the compiler error....
also I wonder if `language_storage` is a good name for this, though something to think about and do in time in another PR